packages/react/src/components/Footer/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Footer } from "./Footer";
+export { default as FooterArgs } from "./Footer.args";
```

Comment:
Hey @wc-lei, this is the source of the rollup error. Rollup will bundle anything in `/components/**/index.ts`, so here it's trying to bundle the Args file which includes an SVG. But we don't want to bundle the Args files, those are only needed for Storybook and for Tests. So instead of exporting it here, just import it directly from the Storybook story or wherever else you need it.